playbooks/collect_instance_statistics/collect_elasticsearch_data.yml

```diff
@@ -0,0 +1,59 @@
+---
+# A simple one-off playbook that gathers basic usage information from the configured logs server.
+# This is used by the `instance_statistics_csv` management command.
+
+- hosts: all
+  vars:
+    # This should be overridden to be more unique.
+    local_output_dir: /tmp/
+    remote_output_filename: /tmp/activity_report
+  tasks:
+    - name: Ensure necessary packages are installed
+      become: yes
+      apt:
+        name: ['python-setuptools', 'virtualenv']
+        state: present
+
+    - name: Copy elasticsearch_requirements.txt
+      copy:
+        src: elasticsearch_requirements.txt
+        dest: /tmp/elasticsearch_requirements.txt
+
+    - name: Install Elasticsearch packages
+      pip:
+        requirements: /tmp/elasticsearch_requirements.txt
+        virtualenv: /home/ubuntu/instance_activity_venv
+        virtualenv_python: python3
+
+    # We have to do a copy+command pair here (instead of script) in order to get
+    #   separate channels for stdout and stderr.
+    # See also: https://github.com/ansible/ansible-modules-core/pull/4000/files
+    - name: Copy query_elasticsearch script
+      copy:
+        src: query_elasticsearch.py
+        dest: /tmp/query_elasticsearch.py
+
+    - name: Run query_elasticsearch script
+      command: >
+        /home/ubuntu/instance_activity_venv/bin/python /tmp/query_elasticsearch.py
```

Comment:
@pcockwell, can you make use of the variable to generate the path to the python interpreter?

Reply:
Done in the next commit